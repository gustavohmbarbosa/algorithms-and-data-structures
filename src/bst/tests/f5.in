1
23
1
80
1
21
1
24
1
214
1
14
1
11
5
99