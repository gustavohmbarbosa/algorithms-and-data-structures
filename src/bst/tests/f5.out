[214][80][24][23][21][14][11]
